src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -459,6 +464,49 @@ void CropWindowTool::plugDirtied( const Gaffer::Plug *plug )
 	}
 }
 
+void CropWindowTool::nodeMetadataChanged( IECore::InternedString key, const Gaffer::Node* node )
+{
+	if( !m_cropWindowPlug || m_overlayDirty )
+	{
+		return;
+	}
+
+	if( node != m_cropWindowPlug->node() )
+	{
+		return;
+	}
+
+	if( MetadataAlgo::readOnlyAffectedByChange( key ) )
+	{
+		m_needCropWindowPlugSearch = m_overlayDirty = true;
+		view()->viewportGadget()->renderRequestSignal()(
+			view()->viewportGadget()
+		);
+	}
+}
+
+void CropWindowTool::plugMetadataChanged( IECore::InternedString key, const Gaffer::Plug* plug )
+{
+	if( !m_cropWindowPlug || m_overlayDirty )
```

Comment:
As above.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3239/commits/558040aa1361bf1df48a471a037f3655bff30c97